### ISSUE TITLE
Instance wont load anymore

### ISSUE THREAD

**Issue by rcewl** [Tyrsis/EssentialsPlugin#78]
when i use path="<user>/dedicatedserver"
it wont load the save/<map> instance
even when i force the instance with the instance option.

it will create a new instance folder with a nummber behind it ?!

**Comment by Pandonic** [comment]
Yeah I have this problem. I've no idea how to tell SE Server Extender to specifically load a map file.
